src/main/java/com/bulkbuy/enterprise/dao/RetrofitClientInstance.java

```diff
@@ -6,7 +6,7 @@
 public class RetrofitClientInstance {
 
     private static Retrofit retrofit;
-    //NEED TO FIND JSON SOURCE
+    //TODO: NEED TO FIND JSON SOURCE
```

Comment:
use special comments when you still need to add logic - this will display a warning to help you not forget to do this
https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3